features/karaf/src/main/feature/feature.xml

```diff
@@ -41,6 +41,7 @@
 
     <bundle start-level="75">mvn:org.eclipse.smarthome.core/org.eclipse.smarthome.core.thing.xml/${project.version}</bundle>
     <bundle>mvn:org.eclipse.smarthome.core/org.eclipse.smarthome.core.transform/${project.version}</bundle>
+    <bundle>mvn:org.eclipse.smarthome.core/org.eclipse.smarthome.core.audio/${project.version}</bundle>
```

Comment:
well, the audio package was already part of smarthome.core bundle, so this is actually not a change, just a refactoring while keeping the status quo. If you have a requirement to split this off the core, feel free to suggest a separate PR.